src/ray/rpc/grpc_server.h

```diff
@@ -35,6 +35,11 @@ class GrpcServer {
   GrpcServer(const std::string &name, const uint32_t port)
       : name_(name), port_(port), is_closed_(true) {}
 
+  GrpcServer(const std::string &name, const std::string &unix_socket_path)
```

Comment:
Document this constructor, and also update the other constructor's documentation.
E.g., `Construct a gRPC server that listens on a TCP port`.